packages/integrations/prefetch/README.md

```diff
@@ -78,6 +80,29 @@ export default defineConfig({
   })],
 });
 ```
+
+### config.intentSelector
+By default, the prefetch script also searches the page for any links that include a `rel="prefetch-intent"` attribute, ex: `<a rel="prefetch-intent" />`. This behavior can be changed in your astro.config.* file to use a custom query selector when finding prefetch-intent links.
```

Comment:
```suggestion
By default, the prefetch script also searches the page for any links that include a `rel="prefetch-intent"` attribute, ex: `<a rel="prefetch-intent" />`. This behavior can be changed in your `astro.config.*` file to use a custom query selector when finding prefetch-intent links.
```
Never let me look at something again, or I'll find something else!  :joy:

Yes, good to go! Sorry, in general it's usually fine if you just applied my suggestions, and I don't need to review again. But, while I'm here, let's inline code the file name here!